gradle/libs.versions.toml

```diff
@@ -32,7 +32,7 @@ conscrypt-android = { module = "org.conscrypt:conscrypt-android", version.ref =
 conscrypt-openjdk = { module = "org.conscrypt:conscrypt-openjdk-uber", version.ref = "org-conscrypt" }
 eclipseOsgi = "org.eclipse.platform:org.eclipse.osgi:3.18.200"
 findbugs-jsr305 = "com.google.code.findbugs:jsr305:3.0.2"
-gradlePlugin-android = "com.android.tools.build:gradle:7.2.2"
+gradlePlugin-android = "com.android.tools.build:gradle:7.4.0-rc03"
```

Comment:
```

Could not determine the dependencies of task ':android-test:lintReportDebug'.
> Could not resolve all dependencies for configuration ':android-test:debugUnitTestRuntimeClasspath'.
   > Could not create task ':android-test:packageDebugAssets'.
      > Cannot use @TaskAction annotation on method IncrementalTask.taskAction$gradle_core() because interface org.gradle.api.tasks.incremental.IncrementalTaskInputs is not a valid parameter to an action method.
```